plugins/usbdmx/AsyncPluginImpl.cpp

```diff
@@ -239,6 +242,14 @@ bool AsyncPluginImpl::NewWidget(Sunlite *widget) {
       new GenericDevice(m_plugin, widget, "Sunlite USBDMX2 Device", "usbdmx2"));
 }
 
+bool AsyncPluginImpl::NewWidget(USBDMXCom *widget) {
+  return StartAndRegisterDevice(
+      widget,
+      // TODO(Someone): Add the serial here like ShowJockey if present)
```

Comment:
Does it have a serial number? Do you need a hand with this?

Reply:
The serial can change from device to device. See https://www.usbdmx.com/building_8.html
If it is for device identification, we can do a case insensitive search for "usbdmx.com" in the manufacturer field?

Reply:
It's just about showing the serial number in the description within OLA. Take a look at the ShowJockey code, it's pretty trivial.

Reply:
See below:

Reply:
I added the widget->SerialNumber(), but I obtain only "USBDMX.com Device (4-20)" .. why "4-20" ?